pandas/tests/indexing/test_floats.py

```diff
@@ -10,6 +10,9 @@
 import pandas.util.testing as tm
 
 
+ignore_ix = pytest.mark.filterwarnings("ignore:\\n.ix:DeprecationWarning")
```

Comment:
maybe move this to the top-level conftest and use it above (a number of files where catching all of the ix warnings)

Reply:
Hmm do you know how to use marks defined in the top-level `conftest.py`? I'm not actually sure that's possible.

Reply:
i think you can just import it